ts/package.json

```diff
@@ -42,7 +51,7 @@
     "format": "prettier --write './**/*.{js,jsx,ts,tsx,css,json}' --config ./.prettierrc",
     "lint": "eslint src",
     "lint:fix": "npm run lint -- --fix",
-    "postbuild": "script/generate-exports.ts",
+    "postbuild": "script/generate-exports.js",
```

Comment:
ts wasn't working?

Reply:
in CI no. I would need to compile or add ts globally. not a big deal but also it has no types so decided to just make it js

Reply:
Ok yea makes sense